### PR TITLE
Add link to github repo of the website

### PR DESCRIPTION
**Pull request by rpannek**
If people would like to contribute to the website it is a nice thing
to have a link back to the source code.

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -16,6 +16,7 @@
 title: PELUX®
 email: info@pelagicore.com
 description: PELUX is a Linux based platform for your automotive project.
```
Comment (zeenix) [inline _config.yml]: I know you're not changing this here but I though I comment here before I forget: "your" sounds way too personal and direct to me. Maybe we just say "for IVI systems"?

File: _includes/footer.html [hunk 2]
```diff
@@ -0,0 +1,49 @@
+<footer class="site-footer">
```
Comment (zeenix) [inline _includes/footer.html]: Is addition of this new file part of "Add link to github repo of the website"? If not, it should go in a separate patch.

Comment (rpannek) [inline _includes/footer.html]: Yes it is, normally the file lives in the gem for this template, but because I need to change this file I am getting it out of the gem and into the repo to be able to change it to add this link to the repo.

**Comment by rpannek** [comment]
Fixed.

File: _includes/footer.html (hunk 2 above)
Comment (rpannek) [inline _includes/footer.html]: Ok, I split it into two commits now.
